darts/tests/models/forecasting/test_local_forecasting_models.py

```diff
@@ -56,7 +56,7 @@
     (StatsForecastAutoARIMA(season_length=12), 4.6),
     (StatsForecastAutoTheta(season_length=12), 5.5),
     (StatsForecastAutoCES(season_length=12, model="Z"), 7.3),
-    (StatsForecastAutoETS(season_length=12, model="AAZ"), 4.1),
+    (StatsForecastAutoETS(season_length=12, model="AAZ"), 7.3),
```

Comment:
just to check, was statsforecast previously wrongly using a different `model` for "AAZ", or was it just a bug in general?
If we the former applies, could we use that other model to achieve the 4.1 error?

Reply:
During the parameter optimization for the placeholder "Z" (seasonality state), the `"AAM"` combination was tested (which is apparently not allowed, I did not investigate why) and they fixed this bug. For the record, with 1.5.0, passing explicitly `"AAM"` would raise an error.

Reply:
sounds good, thanks!